src/redis-cli.c

```diff
@@ -3264,6 +3264,8 @@ static int issueCommandRepeat(int argc, char **argv, long repeat) {
                 config.cluster_reissue_command = 0;
                 return REDIS_ERR;
             }
+            config.dbnum = 0;
```

Comment:
@vitahlin why reset it to 0? may need some comments.

Reply:
what about `Reset dbnum after reconnecting so we can re-select the previous db in cliSelect()`.